data/json/recipes/armor/feet.json

```diff
@@ -1320,6 +1343,13 @@
     "proficiencies": [ { "proficiency": "prof_basketweaving" } ],
     "components": [ [ [ "straw_pile", 4 ], [ "birchbark", 6 ] ] ]
   },
+  {
+    "result": "xl_straw_sandals",
+    "type": "recipe",
+    "copy-from": "straw_sandals",
+    "time": "2 h",
```

Comment:
same comment, bigger sandals means more work here

Reply:
I've increased the craft time roughly proportionally to the increased crafting materials.